README.md

```diff
@@ -1 +1,97 @@
-# cron-command
+# EasyEngine/cron-command
+
+Manages cron jobs in EasyEngine
+
+`cron` command contains following subcommand
+ * [add](#add)
+ * [update](#update)
+ * [list](#list)
+ * [delete](#delete)
+ * [run-now](#run-now)
+ 
+ ## add
+ 
+ Adds a cron job to run a command at specific interval etc.
+
+ ```
+ # Adds a cron job on example.com every 10 minutes
```

Comment:
We should give one more example explaining traditional cron schedule is also supported. For example, for every one minute `* * * * *`

Reply:
Good idea. Done